.goreleaser.yaml

```diff
@@ -0,0 +1,25 @@
+project_name: in-toto
+builds:
+  - ldflags:
+      - "-s -w"
+      - "-extldflags=-zrelro"
+      - "-extldflags=-znow"
+      - "-X cmd.tag={{.Version}}"
+      - "-X cmd.commit={{.FullCommit}}"
+      - "-X cmd.date={{.CommitDate}}"
+    env:
+      - "CGO_ENABLED=0"
+      - "GO111MODULE=on"
+      - "GOFLAGS=-mod=readonly -trimpath"
+    goos:
+      - linux
+      - darwin
+      - windows
+    goarch:
+      - amd64
+    main: ./
+signs:
+  - cmd: cosign
+    signature: "${artifact}.sig"
+    args: ["sign-blob", "-oidc-issuer=https://token.actions.githubusercontent.com", "-output=${signature}", "${artifact}"]
```

Comment:
make it compatible with the new flag standard by using two dashes, WDYT?

Reply:
two dashes?

Reply:
--oidc-issuer and --output.

Reply:
@developer-guy  done.